read.go

```diff
@@ -156,24 +156,31 @@ func viewLocalTimeline(app *App, w http.ResponseWriter, r *http.Request) error {
 	return showLocalTimeline(app, w, r, page, vars["author"], vars["tag"])
 }
 
-func updateTimelineCache(tl *localTimeline) {
-	// Fetch posts if enough time has passed since last cache
-	if tl.posts == nil || tl.m.Invalidate() {
+// updateTimelineCache will reset and update the cache if it is stale or
+// the boolean passed in is true.
+func updateTimelineCache(tl *localTimeline, reset bool) {
+	if reset {
+		tl.Reset()
```

Comment:
Getting this error when trying to compile:

```
../../read.go:163:5: tl.Reset undefined (type *localTimeline has no field or method Reset)
```

Reply:
Ahh yes, sorry about that